sdk/storage/storage-file-share/test/perfstress/track-2/storageTest.spec.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PerfStressTest } from "@azure/test-utils-perfstress";
+
+import { ShareClient, ShareDirectoryClient, ShareServiceClient } from "../../../src";
+
+// Expects the .env file at the same level as the "test" folder
+import * as dotenv from "dotenv";
+dotenv.config();
+
+export abstract class StorageFileShareTest<TOptions> extends PerfStressTest<TOptions> {
+  shareServiceClient: ShareServiceClient;
+  shareClient: ShareClient;
+  directoryClient: ShareDirectoryClient;
+  static shareName = `newshare${new Date().getTime()}`;
+  static dirName = `newdirectory${new Date().getTime()}`;
+
+  constructor() {
+    super();
+    this.shareServiceClient = ShareServiceClient.fromConnectionString(
+      StorageFileShareTest.getEnvVar("STORAGE_CONNECTION_STRING")
+    );
+    this.shareClient = this.shareServiceClient.getShareClient(StorageFileShareTest.shareName);
+    this.directoryClient = this.shareClient.getDirectoryClient(StorageFileShareTest.dirName);
+  }
+
+  public async globalSetup() {
```

Comment:
Missing calls to super setup and cleanup

Reply:
This is the base class that extends `PerfStressTest`, in which `globalSetup` and the other methods aren't implemented.
Hence, not required.